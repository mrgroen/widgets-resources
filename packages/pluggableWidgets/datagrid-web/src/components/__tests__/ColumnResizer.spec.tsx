import { shallow } from "enzyme";
import { createElement } from "react";
import { ColumnResizer } from "../ColumnResizer";

describe("Column Resizer", () => {
    it("renders the structure correctly", () => {
        const component = shallow(<ColumnResizer setColumnWidth={jest.fn()} />);

        expect(component).toMatchSnapshot();
    });
});
