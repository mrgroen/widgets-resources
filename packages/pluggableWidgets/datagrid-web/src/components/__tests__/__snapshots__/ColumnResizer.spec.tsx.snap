// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column Resizer renders the structure correctly 1`] = `
<div
  className="column-resizer"
  onMouseDown={[Function]}
  onTouchStart={[Function]}
>
  <div
    className="column-resizer-bar"
  />
</div>
`;
